src/services/study/network-modifications.ts

```diff
@@ -960,14 +961,10 @@ export function createLine({
             busOrBusbarSectionId1: busOrBusbarSectionId1,
             voltageLevelId2: voltageLevelId2,
             busOrBusbarSectionId2: busOrBusbarSectionId2,
-            currentLimits1: {
-                permanentLimit: permanentCurrentLimit1,
-                temporaryLimits: temporaryCurrentLimits1,
-            },
-            currentLimits2: {
-                permanentLimit: permanentCurrentLimit2,
-                temporaryLimits: temporaryCurrentLimits2,
-            },
+            [OPERATIONAL_LIMITS_GROUPS_1]: limitsGroups1,
+            [OPERATIONAL_LIMITS_GROUPS_2]: limitsGroups2,
```

Comment:
Please try to write homogeneous code
```suggestion
            operationalLimitsGroups1: limitsGroups1,
            operationalLimitsGroups2: limitsGroups2,
```

Reply:
As you wish.